docs/asyncapi.yaml

```diff
@@ -694,4 +694,4 @@ components:
     Period:
```

Comment:
`asyncapi` is missing an entry for a schedule, needs added.

This should be a good guide: https://github.com/DEFRA/ffc-pay-statement-generator/blob/152263/stop-schedules-publishing/docs/asyncapi.yaml

Reply:
Added schedule as entry, whole `asyncapi` needs review, hence added review of  `asyncapi`  to backlog.